tutorials/W2D5_ClimateResponse-AdaptationImpact/W2D5_Tutorial1.ipynb

```diff
@@ -1250,9 +803,263 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.18"
+   "version": "3.11.5"
+  },
+  "widgets": {
+   "application/vnd.jupyter.widget-state+json": {
```

Comment:
This should be python 3.9. Our build pipeline is currently using py3.9. Please create a local env for python 3.9 and make sure everything still works. You should be clearing the kernel and outputs before saving.

Reply:
this is still not resolved